.npm/execute.js

```diff
@@ -0,0 +1,37 @@
+const spawn = require("child_process").spawn;
+
+export async function execute(binPath, args) {
+  const cca = spawn(binPath, args);
+
+  cca.on("error", error => {
+    console.error("Could not execute create-comit-app:", error);
+  });
+
+  async function handleSignal(code) {
```

Comment:
This seems to be unnecessarily `async`, you are not `await`-ing in this function.

Reply:
Yes we are line 15, 19, 23.

Reply:
I was taking about _this_ function. The function `handleSignal` is one line long and it does not use `await`. Hence the `async` is unnecessary. Did you maybe mean to _return_ the return value of `cca.kill`?

Reply:
`kill` does not return a Promise: https://nodejs.org/api/child_process.html#child_process_subprocess_kill_signal

I think you can safely remove the `async` here :)